app/views/content/blog/why-i-became-a-teacher.md

```diff
@@ -18,7 +18,6 @@ keywords:
 tags:
   - becoming a teacher
   - advisers
-closing_paragraph: enriching-the-lives-of-young-people
 ---
```

Comment:
@gemmadallmandfe the first paragraph of this blog post seems really strange - should we be talking about TTAs here when the post is called 'why I became a teacher'? Wouldn't a user expect an introduction to whoever is writing the blog, not to be told what TTAs do?

Reply:
Agree! Suggest we remove that first para - the second para feels more like an intro?